btr-web/btr-common-components/components/bcros/inputs/DateSelect.vue

```diff
@@ -45,9 +46,8 @@ const updateDate = (val: Date | null) => {
   selectedDate.value = val
 }
 
-const selectedDateDisplay: ComputedRef<string> = computed(() => selectedDate.value?.toLocaleDateString() || '')
+const selectedDateDisplay: ComputedRef<string> = computed(
+  () => selectedDate.value ? moment(selectedDate.value).format('YYYY-MM-DD') : ''
```

Comment:
this is actually going against the design :P , can you change it back?

Reply:
updated this due to changes in design.